app/models/street/StreetEdgeTable.scala

```diff
@@ -199,6 +199,48 @@ object StreetEdgeTable {
     }
   }
 
+
+  /**
+    * Calculates the distance audited today by all users
+    * 
+    * Author: Aroosh Kumar
+    * Date: 4/24/2020
+    * 
+    * @return The distance audited today by all users in miles
+    */
+  def auditedStreetDistanceToday(): Float = db.withSession { implicit session =>
+    val getDistanceQuery = Q.queryNA[Float](
+      """SELECT ST_Length(ST_Transform(geom, 26918))
+        |FROM sidewalk.street_edge
+        |INNER JOIN sidewalk.audit_task ON (audit_task.task_end AT TIME ZONE 'PST')::date = (now() AT TIME ZONE 'PST')::date
```

Comment:
So I think the way you want to reorganize this query is to include only the part that joins the two tables in the `INNER JOIN ... ON`. Then any other filters you are using would go in a `WHERE` clause. So it would look more like:
```
SELECT ST_Length(ST_Transform(geom, 26918))
FROM street_edge
INNER JOIN audit_task ON street_edge.street_edge_id = audit_task.street_edge_id
WHERE street_edge.deleted = FALSE
    AND audit_task.completed = TRUE
    AND ...
```